src/hotspot/os_cpu/aix_ppc/atomicAccess_aix_ppc.hpp

```diff
@@ -26,392 +26,8 @@
 #ifndef OS_CPU_AIX_PPC_ATOMICACCESS_AIX_PPC_HPP
 #define OS_CPU_AIX_PPC_ATOMICACCESS_AIX_PPC_HPP
 
-#ifndef PPC64
-#error "Atomic currently only implemented for PPC64"
-#endif
+// Included in atomicAccess.hpp header file.
 
```

Comment:
Maybe add comment like "We use inline assembler which is shared for several PPC64 platforms."

Reply:
Agree.. good time to update the comment and make it more precise.